### DISCUSSION TITLE
What causes all of my ha-websocket nodes to disappear as not installed??

### DISCUSSION THREAD

**Discussion by rlust** [zachowj/node-red-contrib-home-assistant-websocket#1675] [Q&A]
I am now getting these messages with my past working node-red flows.
Flows stopped due to missing node types.
If I go to the pallet I see this for the HA-websocket install.

node-red-contrib-home-assistant-websocket
0.74.2
[object Object]

server
ha-device-config
ha-entity-config
ha-api
api-call-service
ha-device
server-state-changed
ha-button
ha-text
api-current-state
server-events
api-render-template
ha-events-calendar
ha-wait-until
ha-sensor

**Comment by zachowj** [comment]
What version of Node-RED are you running?

**Reply by rlust** [reply]
Node-red v3.1.3

**Comment by rlust** [comment]
Still happening
